pkg/metrics/data/daemon.go

```diff
@@ -20,4 +21,11 @@ var (
 		},
 		[]string{nydusdEventLabel},
 	)
+	NydusdVersion = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "nydusd_version_string",
```

Comment:
It would be better to rename the metric name to "nydusd_count", version is just is label of the metircs.